packages/profiling-node/scripts/prune-profiler-binaries.js

```diff
@@ -0,0 +1,192 @@
+#! /usr/bin/env node
+const fs = require('fs');
+
+let SOURCE_DIR, PLATFORM, ARCH, STDLIB, NODE, HELP;
+
+for (let i = 0; i < process.argv.length; i++) {
+  const arg = process.argv[i];
+  if (arg.startsWith('--target_dir_path=')) {
```

Comment:
Depending on minimum supported Node.js target we can use util.parseArgs: https://nodejs.org/api/util.html#utilparseargsconfig

Reply:
We could do that, but I'd rather have this keep working for folks on <16.17. We don't prebuild binaries for those node versions, however building from source still works and it'd be annoying for users if we broke it here. We can probably do this once 22 is LTS and 🤞🏼 nobody is running v14 anymore